image.go

```diff
@@ -60,6 +71,14 @@ func (img *Image) Copy() *Image {
 	return newImage(C.cvCloneImage(img.iplImage), img.exif)
 }
 
+func Validate(r io.Reader) (err error) {
```

Comment:
We may want to call this method on image upload to ensure that bad images aren't being stored.